lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journeys/dcmaw-driving-licence.yaml

```diff
@@ -0,0 +1,112 @@
+name: DCMAW and Driving Licence
+description: >-
+  The combined journey for DCMAW and Driving Licence CRIs.
+entryEvents:
+  next:
+    targetState: CRI_DCMAW
+  mitigation-dcmaw:
+    targetState: MITIGATION_01_CRI_DCMAW
+nestedJourneyStates:
+  CRI_DCMAW:
+    response:
+      type: cri
+      criId: dcmaw
+    parent: CRI_STATE
+    events:
+      next:
+        targetState: POST_DCMAW_SUCCESS_PAGE
+      dl-auth-source-check:
+        targetState: CRI_DRIVING_LICENCE_AUTH_SOURCE_CHECK
+      not-found:
+        exitEventToEmit: next-multiple-doc
```

Comment:
I'm not sure about these event names - IMO `next` should be reserved for successful completion of the check.

Thinking about all the possibilities, I could imagine the exit states being something like:
- `next` - Completed the DCMAW doc check (ready to move onto address and fraud)
- `incomplete` (or something equivalent) - did not complete the doc check, but did not acquire any CIs (dropout, a problem with the device, temporary unavailability etc.)
    - If useful, this could be refined further to distinguish the sub-cases
- `incomplete-invalid-dl` - Dropped out at the DL auth source check (e.g. because of OCR misread)
- `alternate-doc-invalid-dl` - Got a DL doc CI after the DL auth source check

In general it should describe what happened in this nested journey, not a prediction about what may/may not come next. They might not need to go to address, if it's part of a name change or mitigation route!